guides/common/modules/proc_migrating-to-external-databases.adoc

```diff
@@ -6,7 +6,7 @@
 Back up and transfer existing data, then use the `{foreman-installer}` command to configure {Project} to connect to an external PostgreSQL database server.
 
 .Prerequisites
-* You have installed and configured a PostgreSQL server on a {EL} server.
+* You have installed and configured a PostgreSQL server on an external server.
```

Comment:
```suggestion
* You have installed and configured a PostgreSQL server on a dedicated server.
```
Nitpick, non-blocking.

Reply:
Dedicated servers often have the connotation of being physical servers. See https://en.wikipedia.org/wiki/Dedicated_hosting_service. I'd like to avoid that terminology because it can be a virtual server too.